cpp/models/ode_secirts/analyze_result.h

```diff
@@ -0,0 +1,214 @@
+/* 
+* Copyright (C) 2020-2024 MEmilio
+*
+* Authors: Wadim Koslow, Daniel Abele, Martin J. Kühn
```

Comment:
@lenaploetzke just spotted that you may need to update the authors for the new files.

Reply:
good point ;) Thank you. Is now updated